queries.sql

```diff
@@ -7,4 +7,45 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Hey there! 🗑️ After deleting animal records within a transaction, please make sure to verify that all records have been successfully removed. 🕵️‍♂️ If confirmed, proceed with the rollback and double-check to ensure that all deleted records have been restored. 🔄 Your meticulous verification ensures the accuracy of our database operations! 👌

```suggestion
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;


```